io/fasta/fasta_test.go

```diff
@@ -11,6 +11,31 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func BenchmarkFastaLegacy(b *testing.B) {
+	var fastas []Fasta
+	for i := 0; i < b.N; i++ {
+		fastas, _ = Read("data/uniprot_1mb_test.fasta")
+	}
+	_ = fastas
+}
+
+func BenchmarkParser(b *testing.B) {
```

Comment:
this benchmark errors out and doesn't hit any code.
![Screen Shot 2022-12-01 at 8 34 13 PM](https://user-images.githubusercontent.com/7042260/205215517-e2480367-8ce9-47d5-81bf-e5925dd3ba3f.png)

Reply:
Yeah, these benchmarks depended on unzipping the gz fasta, my bad. Fixing now!